kotsadm/pkg/handlers/download.go

```diff
@@ -24,7 +24,7 @@ func DownloadApp(w http.ResponseWriter, r *http.Request) {
 	a, err := app.GetFromSlug(r.URL.Query().Get("slug"))
 	if err != nil {
 		logger.Error(err)
-		w.WriteHeader(500)
```

Comment:
how about a `if err == sql.ErrNoRows ` check

Reply:
yeah, with `errors.Is`
makes sense